myapp/spec/controllers/articles_controller_spec.rb

```diff
@@ -0,0 +1,155 @@
+require 'rails_helper'
+
+RSpec.describe ArticlesController, type: :controller do
+  describe "GET #index" do
+    before :each do
+      get :index
+    end
+    it "populates an array of articles" do
+      article = FactoryGirl.create(:article)
+      expect(assigns(:articles)).to eq([article])
+    end
+
+    it "responds successfully with an HTTP 200 status code" do
+      expect(response).to be_success
+      expect(response).to have_http_status(200)
+    end
+
+    it "renders the index view" do
+      expect(response).to render_template("index")
+    end
+  end
+
+  describe "GET #show" do
+    let(:article) { FactoryGirl.create(:article) }
```

Comment:
If you add something in the configuration you can avoid adding the `FactoryGirl` everywhere ;)